google/logging/v2/logging_gapic.yaml

```diff
@@ -352,8 +352,11 @@ interfaces:
     bundling:
       thresholds:
         element_count_threshold: 100
-        request_byte_threshold: 1024 # 1 Kb
+        request_byte_threshold: 20480 # 20 Kb
```

Comment:
Ahh, right. It occurs to me that we don't actually have a place that can enforce message size limits. The bundling thresholds are to trigger bundling and don't throw errors if they are exceeded. 

But one thing we should at least do is not throw exceptions if we are capable of handling a larger message - e.g. grpc can handle 4MB, so the flow controller should be allowed to store that much at least.